multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -208,6 +214,61 @@ def is_imagepath_column(
         return False
 
 
+def is_imagebytearray_column(
+    data: pd.Series,
+    col_name: str,
+) -> bool:
+    """
+    Identify if a column contains valid image bytearray.
+
+    Parameters
+    ----------
+    data
+        One column of a multimodal pd.DataFrame for training.
+    col_name
+        Name of column.
+
+    Returns
+    -------
+    Whether the column is an image-bytearray column.
+    """
+    sample_num = min(len(data), 500)
```

Comment:
Can we make the `500` as a configurable variable?

Reply:
Sure, updated!